geomesa-hbase/geomesa-hbase-tools/src/main/scala/org/locationtech/geomesa/hbase/tools/ingest/HBaseBulkLoadCommand.scala

```diff
@@ -49,7 +50,11 @@ class HBaseBulkLoadCommand extends HBaseDataStoreCommand {
     val locator = ds.connection.getRegionLocator(tableName)
     val config = new Configuration
     config.set("hbase.loadincremental.validate.hfile", params.validate.toString)
-    new LoadIncrementalHFiles(config).doBulkLoad(input, ds.connection.getAdmin, table, locator)
+    // use reflection to get around 1.4 vs 2.2 API differences (method return type void vs Map)
+    val load = new LoadIncrementalHFiles(config)
+    val method =
+      load.getClass.getMethod("doBulkLoad", classOf[Path], classOf[Admin], classOf[Table], classOf[RegionLocator])
+    method.invoke(load, input, ds.connection.getAdmin, table, locator)
```

Comment:
I'm assuming you tested this?  (Otherwise, noticing the difference would be tough:)....)

Reply:
yeah, the method is the same so there was no compiler error, but the return value is different so it breaks at runtime